### PR TITLE
Ditto MQTT self signed ssl example

### PR DESCRIPTION
**Pull request by bernar0507**
There is no tutorial for the MQTT SSL for Ditto. 
I made this to help people who are starting out and want to have a secure connection between their physical thing and the Digital twin.

### CODE REVIEW

File: mqtt-iwatch-ssl/README.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
+# Eclipse-Ditto-MQTT-iWatch-SSL
+This example presents how to configure Ditto to be able update things via MQTT with SSL. We will create an iWatch from a WoT TM (Web of Things Thing Model) in this example. Our Digital Twin will be updated via MQTT, using synthetic data.
+In this tutorial, we will ensure data security in traffic with two-factor authentication (`Username and Password` + `Certificates`).
```
Comment (thjaeckle) [inline mqtt-iwatch-ssl/README.md:3]: Are you sure that really both authentication mechanisms are used at the same time?
I would expect that only the client certificate is used.
I would not expect that the mqtt broker supports 2 factor auth in that way..

**Review by thjaeckle — CHANGES_REQUESTED** [review]
The introduction needs some clarification IMO

File: mqtt-iwatch-ssl/README.md (hunk 1 above)
Comment (bernar0507) [reply mqtt-iwatch-ssl/README.md]: I changed the wording. I think that dual authentication is more accurate.

File: mqtt-iwatch-ssl/README.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
+# Eclipse-Ditto-MQTT-iWatch-SSL
+This example presents how to configure Ditto to be able update things via MQTT with SSL. We will create an iWatch from a WoT TM (Web of Things Thing Model) in this example. Our Digital Twin will be updated via MQTT, using synthetic data.
```
[the remark this answers is not in the document]
Comment (bernar0507) [reply mqtt-iwatch-ssl/README.md]: I've rewrote the intro and gave more detail. I think its better now.

**Review by thjaeckle — APPROVED** [review]
LGTM, thanks for the example and the effort in documenting it 👍

**Pull request merged**